### ISSUE TITLE
Option to automatically close original tab

### ISSUE THREAD

**Issue by neoOpus** [aloneguid/bt#32]
Hi,

I thought that adding an option to set it to close the original tab/windows after it opens it in another browser (or maybe same but different profile or even window), would be interesting to have since it reduces interaction, resource usage etc.

**Comment by aloneguid** [comment]
Good idea. From browser extension?

**Comment by neoOpus** [comment]
Yes it could be a communication between both BT and browser extensions, when the one on the receiving browser detects that the tabs has been opened then it sends a message back to BT that will relay it to the extension in the original browser to order the closing of the tab, when done this way and being optional it is safe (by that I mean no tabs will be lost in transfer) and it will reduce annoyance, anxiety and resource usage as I stated earlier.

**Comment by AJolly** [comment]
@neoOpus I use this extension(s) extensively to send tabs to BT / close existing tab, etc.  It's very configurable. 
 https://webextension.org/listing/open-in.html

**Comment by aloneguid** [comment]
This is done in the latest version of all the extensions. Closing toolbar button will close current tab after opening in BT.
